src/oas.md

```diff
@@ -1424,16 +1449,30 @@ components:
   examples:
     LogJSONSeq:
       summary: Log entries in application/json-seq
+      dataValue: [
+        {
+          "timestamp": "1985-04-12T23:20:50.52Z",
+          "level": 1,
+          "message": "Hi!"
+        },
+        {
+          "timestamp": "1985-04-12T23:20:51.37Z",
+          "level": 1,
+          "message": "Bye!"
+        }
+      ]
       # JSON Text Sequences require an unprintable character
       # that cannot be escaped in a YAML string, and therefore
       # must be placed in an external document shown below
-      externalValue: examples/log.json-seq
+      externalSerializedValue: examples/log.json-seq
```

Comment:
It would be really cool if we had the html renderer detect that this keyword and add a hyperlink to a real file named "./examples/log.json-seq" in the github-pages repo.